src/views/Project/index.jsx

```diff
@@ -117,8 +108,20 @@ export default class Project extends Component {
             Bugs & Issues
           </Typography>
         </header>
-        <TasksTable items={data} />
+        {error && <ErrorPanel error={error} />}
+        {loading && <Spinner />}
+        {!error && !loading && <TasksTable items={data} />}
       </Fragment>
     );
   }
 }
+
+const ProjectClient = props => (
```

Comment:
Only use a single component per file. This can actually be put inside the `render` of `Project`.

Also, the fact that you are deriving this data not from props shows me this could still be used from the decorator, but we can return to that in the future.

Reply:
if I put in the Project, where should i parse the data i got?

Reply:
I realized i need the props to know which file i should open and create the query based on it. thats why I use the Query component instead of @graphql decorator